starfish/__init__.py

```diff
@@ -22,6 +22,9 @@
 # NOTE: if we move to python 3.7, we can produce this value at call time via __getattr__
 __version__ = pkg_resources.require("starfish")[0].version
 
+# Variable to be set by release process
```

Comment:
I would make it super explicit, e.g., `is_released_version` or something like that.

also, cc: #761